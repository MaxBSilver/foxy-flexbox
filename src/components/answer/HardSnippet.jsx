import React from "react";

export default function HardSnippet(props) {
  return (
    <div className="input-three">
      <p className="code-prompt"> flex-wrap : </p>
      <input
        type="text"
        className="search-input"
        name="fwVal"
        value={props.fwVal}
        onChange={props.handleChange}
        placeholder=""
      />
    </div>
  );
}
