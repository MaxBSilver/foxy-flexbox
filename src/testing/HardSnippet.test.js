import React from 'react';
import HardSnippet from '../components/answer/HardSnippet';
import { shallow } from 'enzyme';
describe('EasySnippet', () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(
        <HardSnippet
        />
      );
    });
    it('should match the snapshot with all data passed in', () => {
        expect(wrapper).toMatchSnapshot();
      });
  });