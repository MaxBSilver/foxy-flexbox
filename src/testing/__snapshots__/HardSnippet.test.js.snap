// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EasySnippet should match the snapshot with all data passed in 1`] = `
<div
  className="input-three"
>
  <p
    className="code-prompt"
  >
     flex-wrap : 
  </p>
  <input
    className="search-input"
    name="fwVal"
    placeholder=""
    type="text"
  />
</div>
`;
